changelog/709.bugfix.rst

```diff
@@ -0,0 +1,4 @@
+pass the ``ProgramData`` environment variable when invoking the internal
+development ``fix-lint`` toolset, fixing ``error setting certificate verify
+locations`` errors when cloning a git repo using git for Windows - by
+@jurko-gospodnetic
```

Comment:
Since this is an internal only change (and not a bug in tox itself) it should not have a bug fix changeling entry

Reply:
yeah, remove the changelog please 👍